### PR TITLE
Makes gorillas drop brains when gibbed / butchered

### PR DESCRIPTION
**Pull request by ShizCalev**
🆑 ShizCalev
tweak: Gorillas will now drop their brain when gibbed or butchered.
/🆑

Sister to/requested change for #30841

If there's another prefered method to approach this, let me know.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm [hunk 1]
```diff
@@ -77,6 +77,13 @@
 /mob/living/simple_animal/hostile/gorilla/CanSmashTurfs(turf/T)
 	return iswallturf(T)
 
+/mob/living/simple_animal/hostile/gorilla/gib(no_brain)
+	if(!no_brain)
+		var/mob/living/brain = new /mob/living/brain(drop_location())
+		if(mind)
+			mind.transfer_to(brain)
```
Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm]: Probably should name the brain before doing this

**Comment by FantasticFwoosh** [comment]
Gorillas don't deserve it, when they make the conversion willfully or accidentally its more or less a permanent change that gives temporary strength at risk of dying forever.

- This sets poor precedent for monkeys not giving brains when they are gibbed like this using harm intent and sharp implements.

**Comment by ShizCalev** [comment]
@FantasticFwoosh monkeys do give brains when gibbed when using harm intent with sharp instruments.

https://i.imgur.com/gcV5fbg.gifv

**Comment by ShizCalev** [comment]
Consistency with other monkies, as well as offering a little mercy to players who were permanently taken out of the round by a single click from a traitor 30 seconds in.

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm [hunk 2]
```diff
@@ -77,6 +77,15 @@
 /mob/living/simple_animal/hostile/gorilla/CanSmashTurfs(turf/T)
 	return iswallturf(T)
 
+/mob/living/simple_animal/hostile/gorilla/gib(no_brain)
+	if(!no_brain)
+		var/mob/living/brain = new /mob/living/brain(drop_location())
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm]: var/mob/living/brain/B = new(drop_location()

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
The real solution is to make gorillas carbon.

**Comment by pubby** [comment]
Porting gorillas to carbon touches a lot of code though. I figured the maintainers wouldn't be interested.

This PR is the next best thing.

**Comment by ExcessiveUseOfCobblestone** [comment]
Making them pseudo-carbons is band-aid af though.

I'm not sure you can even examine what they have inhands yet.

**Pull request merged**